tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -297,10 +305,6 @@ class MklDnnConvUtil {
     // Now handle padding. MKL-DNN uses asymetric padding.
```

Comment:
Please also remove these now-orphaned comment lines

Reply:
Thank for the suggestion of removing the duplicate edge.
I took the 1st suggestion.
I think with small change we can achieve this. We can have the following : 
  for (const Edge* e : pred->in_edges()) {
    if (e->IsControlEdge()) {
      //Don't allow duplicate edge
      (*g)->AddControlEdge(e->src(), new_node, false);
    }    
  }
Here, if duplicate edge is found, then it will return nullptr, otherwise it will add the edge.

The addControlEdge is in : https://github.com/tensorflow/tensorflow/blob/01267485b63d440eb9f3b4061d8e8ceaad5bca79/tensorflow/core/graph/graph.cc#L454 

And the AddEdge is in : https://github.com/tensorflow/tensorflow/blob/425b62a344f18c875f6f024b36ae37749cb00feb/tensorflow/core/graph/graph.cc#L403

Please let me know if this looks ok or not.